rimage/file_simple.c

```diff
@@ -497,7 +497,7 @@ int write_logs_dictionary(struct image *image)
 
 			fprintf(stdout, "logs dictionary: size %u\n",
 				header.data_length + header.data_offset);
-			fprintf(stdout, "including fw version of size: %lu\n\n",
+			fprintf(stdout, "including fw version of size: %lu\n",
```

Comment:
Cleanup commit.

Reply:
It is not a cleanup, rather a consequence of adding another printouts by the same patch. Assume it was intentionally added after the last printout in the original implementation.

Reply:
Ah okay.